dev/drivers/scripts/prep/nwps/jevs_nwps_wave_grid2obs_prep.sh

```diff
@@ -0,0 +1,74 @@
+#PBS -N jevs_nwps_wave_grid2obs_prep
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=01:00:00
```

Comment:
Should this be set to 15 minutes to match what is in ecf/scripts/prep/nwps/jevs_nwps_wave_grid2obs_prep.ecf and 
ecf/defs/evs-nco.def ?